src/App.tsx

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    key: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.pressedKey);
+  }
+
+  pressedKey = (event: KeyboardEvent) => {
```

Comment:
```suggestion
  handleKeyPressed= (event: KeyboardEvent) => {
```
![image](https://user-images.githubusercontent.com/96409563/217056439-2cecbfff-763f-4840-8bff-fd29177a69c5.png)